go/libraries/doltcore/table/inmem_table.go

```diff
@@ -144,6 +144,10 @@ func (rd *InMemTableReader) GetSchema() schema.Schema {
 	return rd.tt.sch
 }
 
+func (rd *InMemTableReader) VerifySchema(outSch schema.Schema) (bool, error) {
```

Comment:
Was getting errors if I didn't also add `VerifySchema` here and to `NomsMapReader`. LMK if they should return something else